git_machete/github.py

```diff
@@ -143,9 +143,16 @@ def __fire_github_api_request(method: str, path: str, token: Optional[str], requ
         raise MacheteException(f'Could not connect to {host}: {e}')
 
 
-def __check_pr_already_created(pull: GitHubPullRequest, pull_requests: List[GitHubPullRequest]) -> Optional[GitHubPullRequest]:
+def find_pr_by_base_and_head(base: str, head: str, pull_requests: List[GitHubPullRequest]) -> Optional[GitHubPullRequest]:
     for pr in pull_requests:
-        if pull.base == pr.base and pull.head == pr.head:
+        if base == pr.base and head == pr.head:
+            return pr
+    return None
+
+
+def find_pr_by_number(no: int, pull_requests: List[GitHubPullRequest]) -> Optional[GitHubPullRequest]:
```

Comment:
Nit, for consistency:
```suggestion
def find_pr_by_number(number: int, pull_requests: List[GitHubPullRequest]) -> Optional[GitHubPullRequest]:
```